setup.py

```diff
@@ -20,7 +20,7 @@
         'License :: OSI Approved :: BSD License',
     ],
     install_requires=[
-        'protobuf>=3.7.1',
+        'protobuf>=3.7.1,<3.20',
```

Comment:
Do you think `<4` would work? This would let us accept subsequent minor versions.

Reply:
I've updated, after a couple further checks. Thanks for asking!